Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.12
+FROM golang:1.13.8-stretch
```

Comment:
Go 1.14 is out, maybe just move to that?

Reply:
Updated golang image  with 1.14.2-stretch